manifests/istio-control/istio-config/templates/validatingwebhookconfiguration.yaml.tpl

```diff
@@ -9,7 +9,6 @@ apiVersion: admissionregistration.k8s.io/v1beta1
 kind: ValidatingWebhookConfiguration
 metadata:
   name: istio-galley
-  namespace: {{ .Release.Namespace }}
```

Comment:
The other namespaces should be removed in this file too.

Reply:
done